tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -0,0 +1,126 @@
+#
+# Common widget google-rtc-aec
+#
+# A generic google-rtc-aec widget. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.google-rtc-aec.N.attribute_name"
+#
+# Usage: this component can be used by instantiating it in the parent object. i.e.
+#
+# 	Object.Widget.google-rtc-aec."N" {
+# 		index		1
+# 		type		"effect"
+#	}
+#
+# Where N is the unique instance number for the google-rtc-aec object within the same alsaconf node.
+Define {
+	AEC_UUID	"a6:a0:80:b7:9f:26:6f:46:b4:77:23:df:a0:5a:f7:58"
+}
+
+Class.Widget."google-rtc-aec" {
+	#
+	# Pipeline ID for the google-rtc-aec widget object
+	#
+	DefineAttribute."index" {}
+
+	#
+	# google-rtc-aec object instance
+	#
+	DefineAttribute."instance" {}
+
+	#include common component definition
+	<include/components/widget-common.conf>
+
+	DefineAttribute."num_audio_formats" {
+		# Token set reference name and type
+		token_ref	"sof_tkn_comp.word"
+	}
+
+	#
+	# google-rtc-aec widget UUID
+	#
+	DefineAttribute."uuid" {
+		type	"string"
+		# Token set reference name and type
+		token_ref	"sof_tkn_comp.uuid"
+	}
+
+	DefineAttribute."cpc" {
```

Comment:
what is cpc? Can we really get some docs on these new formats? its really frustrating trying to validate this against out code without context. There is nothing in the docs website or the existing usages.

Reply:
It is described in my rimage documents. CPC : cycle per chunk data (1ms in most case)

Reply:
Are those docs published anywhere?

Reply:
yes,  In rimage doc https://github.com/thesofproject/sof-docs/pull/451